### DISCUSSION TITLE
大世界选了月初开荒，不知道不停的打开关闭窗口

### DISCUSSION THREAD

**Discussion by a3318375** [LmeSzinc/AzurLaneAutoScript#2928] [Q&A]
![image](https://github.com/LmeSzinc/AzurLaneAutoScript/assets/10102582/9815435c-4866-42e0-9ad0-c4b8e57fe3f9)
我没去干涉它，开荒到如图5-2这个图了
原本还有26行动力的，这个图消耗为0，然后它就在不停的打开关闭这个图的小窗。
我手动补充到了47还是在不停的打开关闭，想请问一下是哪个地方出了问题？

INFO     13:42:35.271 │           globe_center: (1313.0, 1132.0)                                         
INFO     13:42:35.274 │ 0.141s      similarity: 0.368                                                    
INFO     13:42:35.276 │ [Globe_center] 156                                                               
INFO     13:42:36.124 │           globe_center: (1313.0, 1132.0)                                         
INFO     13:42:36.127 │ 0.152s      similarity: 0.360                                                    
INFO     13:42:36.130 │ [Globe_center] 156                                                               
INFO     13:42:36.138 │ Click ( 508,  424) @ ZONE_159                                                    
INFO     13:42:37.331 │           globe_center: (1313.0, 1132.0)                                         
INFO     13:42:37.334 │ 0.145s      similarity: 0.398                                                    
INFO     13:42:37.337 │ [Globe_center] 156                                                               
INFO     13:42:38.175 │           globe_center: (1313.0, 1132.0)                                         
INFO     13:42:38.179 │ 0.144s      similarity: 0.357                                                    
INFO     13:42:38.181 │ [Globe_center] 156                                                               
INFO     13:42:39.035 │           globe_center: (1313.0, 1132.0)                                         
INFO     13:42:39.038 │ 0.147s      similarity: 0.353                                                    
INFO     13:42:39.041 │ [Globe_center] 156                                                               
WARNING  13:42:39.049 │ Too many click for a button: ZONE_159                                            
WARNING  13:42:39.052 │ History click: ['ZONE_159', 'ZONE_159', 'ZONE_159', 'ZONE_159', 'ZONE_159',      
         'ZONE_159', 'ZONE_159', 'ZONE_159', 'ZONE_159', 'ZONE_159', 'ZONE_159', 'ZONE_159', 'ZONE_159', 
         'ZONE_159', 'ZONE_159']                                                                         
ERROR    13:42:39.055 │ GameTooManyClickError: Too many click for a button: ZONE_159    
尝试人工干涉手动点进去，又不断重启，日志如下
WARNING  13:58:15.267 │ Zone init timeout                                                                
WARNING  13:58:15.270 │ Unable to get zone name, get current zone from globe map instead                 
INFO     13:58:15.283 │ Click (1204,  670) @ MAP_GOTO_GLOBE

**Comment by stephensund** [comment]
![图片](https://github.com/LmeSzinc/AzurLaneAutoScript/assets/47056935/9d50b0c6-7569-4455-b580-0f47bd18af40)
其实作者已经写提示了，你需要先把主线打完

**Reply by a3318375** [reply]
懂了，我一直以为我打过主线了。因为不怎么进大世界，没看到主线的标记
